python/taichi/lang/exception.py

```diff
@@ -1,3 +1,6 @@
+from taichi._lib import core
```

Comment:
How about `from taichi._lib.core import TaichiTypeError`?

Reply:
It seems that I can't do that... maybe because `taichi._lib.core` is a dynamic lib?